src/main/java/com/exasol/udfstartuptimeimprover/UdfStartUpTimeImprover.java

```diff
@@ -0,0 +1,171 @@
+package com.exasol.udfstartuptimeimprover;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.*;
+import java.util.concurrent.TimeoutException;
+
+import com.exasol.*;
+import com.exasol.bucketfs.*;
+import com.exasol.errorreporting.ExaError;
+
+/**
+ * This class is the entrypoint of the java startup time improver UDF.
+ */
+public class UdfStartUpTimeImprover {
+    /** Name for the UDF parameter that contains the definition of the UDF / adapter script to optimize. */
+    public static final String PARAMETER_UDF_DEFINITION = "UDF_DEFINITION";
+    /** Name for the UDF parameter that contains the path in BucketFS to a file with the classes to preload. */
+    public static final String PARAMETER_CLASSES = "CLASSES";
+    /**
+     * Name of the UDF parameter that connections the name of a connection that stores the write-password of the
+     * BucketFS as 'IDENTIFIED BY'.
```

Comment:
```suggestion
     * Name of the UDF parameter that stores the write-password of the
     * BucketFS as 'IDENTIFIED BY'.
```

Reply:
That does not fit. It's the name of a connection definition. 
How about: 
```suggestion
     * Name of the UDF parameter that stores the name of a connection that stores the write-password of the
     * BucketFS as 'IDENTIFIED BY'.
```

Reply:
👍